prisma/readme.md

```diff
@@ -0,0 +1,5 @@
+### DB migration
+
+```
+cargo run --bin prisma -- migrate dev
+```
```

Comment:
it's better if leave more detailed configuration info

Reply:
yup
i agree
i mean that